README.md

```diff
@@ -46,7 +46,7 @@ target '<Your Target Name>' do
     # optional: for using the AWS service
     pod 'StitchSDK/StitchAWSService', '= 5.1.0'
     # optional: for using the Firebase Cloud Messaging service
-    pod 'StitchSDK/StitchFCMService', '= 5.1.0'
+    pod 'StitchFCMService', '= 5.1.0'
```

Comment:
[nit] I don't think this should have changed, is this actually how it works now?